tests/integrations/basic-fail/Cargo.stdout

```diff
@@ -132,6 +135,135 @@ full stdout:
+
+error: actual output differed from expected
+Execute `DO NOT BLESS. These are meant to fail` to update `tests/actual_tests/bad_pattern_multiple.stderr` to the actual output
+--- tests/actual_tests/bad_pattern_multiple.stderr
++++ <stderr output>
+ error[E0308]: mismatched types
+  --> tests/actual_tests/bad_pattern_multiple.rs:4:9
+... 11 lines skipped ...
+ 
+ error[E0308]: mismatched types
+-  --> tests/actual_tests/bad_pattern_multiple.rs:6:9
++ --> tests/actual_tests/bad_pattern_multiple.rs:6:9
+   |
+ 6 |     add("42", 3);
+... 3 lines skipped ...
+   |
+ note: function defined here
+-  --> $DIR/tests/integrations/basic-fail/src/lib.rs:LL:CC
++ --> $DIR/tests/integrations/basic-fail/src/lib.rs:LL:CC
+   |
+ 1 | pub fn add(left: usize, right: usize) -> usize {
+   |        ^^^
+ 
+-    error: aborting due to 2 previous errors
++error: aborting due to 2 previous errors
+ 
+ For more information about this error, try `rustc --explain E0308`.
+
+Full unnormalized output:
+error[E0308]: mismatched types
+ --> tests/actual_tests/bad_pattern_multiple.rs:4:9
+  |
+4 |     add("42", 3);
+  |     --- ^^^^ expected `usize`, found `&str`
+  |     |
+  |     arguments to this function are incorrect
+  |
+note: function defined here
+ --> $DIR/tests/integrations/basic-fail/src/lib.rs:LL:CC
+  |
+1 | pub fn add(left: usize, right: usize) -> usize {
+  |        ^^^
+
+error[E0308]: mismatched types
+ --> tests/actual_tests/bad_pattern_multiple.rs:6:9
+  |
+6 |     add("42", 3);
+  |     --- ^^^^ expected `usize`, found `&str`
+  |     |
+  |     arguments to this function are incorrect
+  |
+note: function defined here
+ --> $DIR/tests/integrations/basic-fail/src/lib.rs:LL:CC
+  |
+1 | pub fn add(left: usize, right: usize) -> usize {
+  |        ^^^
+
+error: aborting due to 2 previous errors
+
+For more information about this error, try `rustc --explain E0308`.
+
+
+error: `miesmätsched types` not found in diagnostics on line 4
+ --> tests/actual_tests/bad_pattern_multiple.rs:5:17
+  |
+5 |     //~^ ERROR: miesmätsched types
+  |                 ^^^^^^^^^^^^^^^^^^ expected because of this pattern
+  |
+
+error: `miesmätsched types` not found in diagnostics on line 6
+ --> tests/actual_tests/bad_pattern_multiple.rs:7:17
+  |
+7 |     //~^ ERROR: miesmätsched types
+  |                 ^^^^^^^^^^^^^^^^^^ expected because of this pattern
+  |
+
+error: there was 1 unmatched diagnostic
+ --> tests/actual_tests/bad_pattern_multiple.rs:4:9
+  |
+4 |     add("42", 3);
```

Comment:
Maybe there need to be multiple per line for there to be just one diagnostic informing us about multiple cases